docs/source/en/using-diffusers/img2img.md

```diff
@@ -475,6 +485,9 @@ upscaler.enable_xformers_memory_efficient_attention()
 
 image_2 = upscaler(prompt, image=image_1, output_type="latent").images[0]
 ```
+<!--
+ValueError: AutoPipeline can't find a pipeline linked to StableDiffusionLatentUpscalePipeline for None
+-->
```

Comment:
Let's replace the `AutoPipeline` here with `StableDiffusionPipeline`

```suggestion
```

Reply:
StableDiffusionPipeline gives another error; so I used StableDiffusionLatentUpscalePipeline and StableDiffusionUpscalePipeline.